packages/framework/ai-collab/src/aiCollabApi.ts

```diff
@@ -182,3 +182,26 @@ export interface TokenLimits {
 	 */
 	readonly outputTokens?: number;
 }
+
+/**
+ * The GenerateTreeEditsResponse interface defines the structure of the response object
+ *
+ * @alpha
+ */
+export interface GenerateTreeEditsResponse {
+	status: "success" | "failure" | "partial-failure";
+	errorMessage?: string;
+	tokensUsed: TokenUsage;
+	diffs?: Diff[];
+}
+
+/**
+ * The Diff interface defines the structure of the response object
```

Comment:
This comment needs a  more detailed explanation, this doesn't explain the concept of what a diff is.